service/src/main/java/org/broadinstitute/listener/relay/inspectors/SamResourceClient.java

```diff
@@ -30,9 +30,11 @@ public Instant checkWritePermission(String accessToken) {
     try {
       var oauthInfo = tokenChecker.getOauthInfo(accessToken);
       if(oauthInfo.expiresAt().isPresent()) {
+
         samClient.setAccessToken(accessToken);
         var resourceApi = new ResourcesApi(samClient);
 
+        logger.info("token "+accessToken);
```

Comment:
Do we want to log the access token? That is sensitive information.

Reply:
we don't..it's just for debugging cuz for some reason things are not working as I'm testing e2e